### DISCUSSION TITLE
Welcome to 42_tests Discussions!

### DISCUSSION THREAD

**Discussion by poechlauerbe** [poechlauerbe/42_tests#1] [Announcements]
## 👋 Welcome!
If you want something to be added to this testing site - just do it here :)

**Comment by BenzThor** [comment]
Philosophers:

Should print some error: ./philo 2 800 200 200 0

Should print some error: ./philo 3 600 200 200 -

Should print some error: ./philo 3 -600 200 200

Should print that the creation fails, but no other error. (ulimit -v 520000 ; valgrind --tool=helgrind ./philo 200 800 200 200)

**Comment by Nikster3k** [comment]
so_long / cub3D:

open .xpm file in text editor and remove some lines to break the xpm file.
Then run program to check if mlx_xpm_file_to_img is protected, it should now fail.
